subsys/shell/Kconfig

```diff
@@ -68,6 +68,31 @@ config SHELL_BACKSPACE_MODE_DELETE
 	  Some terminals send code: 0x08 (backspace) other 0x7F (delete). When
 	  this option is set shell will expect 0x7F for backspace key.
 
+choice SHELL_NEW_LINE_CHARACTER
+	prompt "Default new line character for shell output"
+
+config SHELL_FLAG_NLCR_DEFAULT
+	bool "Do not map NL or CR"
+	help
+	  Do not map NL or CR. Shell will send send literally all requested data
+	  to the terminal without any modification.
+
+config SHELL_FLAG_ONLCR
```

Comment:
I wonder if this \n to \r\n mapping could be made the default, as that feels the natural default here.